app/models/chat_channel.rb

```diff
@@ -54,22 +79,6 @@ def remove(user)
     end
   end
 
-  def open?
```

Comment:
Removed all those methods as they’re already defined by the `enum` keyword from AR.